_summaries/goodfellow2013maxout_2.md

```diff
@@ -1,19 +1,31 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: Maxout Networks
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 5
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+* What is the core idea? <br/>
+This paper comes up with a new function -- maxout function. Using Maxout function together with dropout techniques can classify images with lowest error rate on several well-known images dataset at that time. 
```

Comment:
I feel that this doesn't adequately describe the core idea.  The fact that it leads to better experimental performance is a consequence of the idea.  I think that the core idea of maxout is to use a _learned_ the activation function that has the following properties:
* It is strictly more expressive than rectifiers (as Fig. 1 shows)
* It produces representations (i.e. layer outputs) that are less sparse, thus alleviating the dead neuron problem.

Reply:
Thanks for your advice, I agree with you that the core idea is not about performance but about maxout function.